roles/disaster_recovery/README.md

```diff
@@ -0,0 +1,93 @@
+oVirt Disaster Recovery
+=========
+
+The `oVirt.disaster-recovery` role responsible to manage the disaster recovery scenarios in oVirt.
+
+Requirements
+------------
+
+ * Ansible version 2.9 or higher
+ * Python SDK version 4.4 or higher
+
+Role Variables
+--------------
+
+| Name                    | Default value         |                                                     |
+|-------------------------|-----------------------|-----------------------------------------------------|
+| dr_ignore_error_clean   | False                 | Specify whether to ignore errors on clean engine setup.<br/>This is mainly being used to avoid failures when trying to move a storage domain to maintenance/detach it.      |
+| dr_ignore_error_recover | True                  | Specify whether to ignore errors on recover.      |
+| dr_partial_import       | True                  | Specify whether to use the partial import flag on VM/Template register.<br/>If True, VMs and Templates will be registered without any missing disks, if false VMs/Templates will fail to be registered in case some of their disks will be missing from any of the storage domains.      |
+| dr_target_host          | secondary             | Specify the default target host to be used in the ansible play.<br/> This host indicates the target site which the recover process will be done.      |
+| dr_source_map           | primary               | Specify the default source map to be used in the play.<br/> The source map indicates the key which is used to get the target value for each attribute which we want to register with the VM/Template.       |
+| dr_reset_mac_pool       | True                  | If True, then once a VM will be registered, it will automatically reset the mac pool, if configured in the VM.        |
+| dr_cleanup_retries_maintenance       | 3                  | Specify the number of retries of moving a storage domain to maintenance VM as part of a fail back scenario.       |
+| dr_cleanup_delay_maintenance       | 120                  | Specify the number of seconds between each retry as part of a fail back scenario.       |
+| dr_clean_orphaned_vms        | True                  | Specify whether to remove any VMs which have no disks from the setup as part of cleanup.       |
+| dr_clean_orphaned_disks        | True                  | Specify whether to remove lun disks from the setup as part of engine setup.       |
+| dr_running_vms		 | /tmp/ovirt_dr_running_vm_list	 | Specify the file path which is used to contain the data of the running VMs in the secondary setup before the failback process run on the primary setup after the secondary site cleanup was finished. Note that the /tmp folder is being used as default so the file will not be available after system reboot.
+
+
+
+Dependencies
+------------
+
+No.
+
```

Comment:
There are global dependencies in the main README.md. You can remove these lines.

Reply:
Done, removed.